tests/monotouch-test/CoreData/NSQueryGenerationTokenTest.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+#if XAMCORE_2_0
+using CoreData;
+using Foundation;
+#else
+using MonoTouch.CoreData;
+using MonoTouch.Foundation;
+#endif
+using NUnit.Framework;
+using Xamarin.Tests;
+
+namespace MonoTouchFixtures.CoreData
+{
+	[TestFixture]
+	[Preserve (AllMembers = true)]
+	public class NSQueryGenerationTokenTest
+	{
+		[Test]
+		public void EncodeWithCoderTest ()
+		{
+			if (!PlatformInfo.Host.IsArch32 ) {
```

Comment:
@spouliot  `PlatformInfo.Host.IsArch32` does the `IntPtr.Size` check so why not use it? just curious 😄